src/components/RepCard/RepCard.css

```diff
@@ -0,0 +1,22 @@
+.rep-image {
+  width: 200px;
+  float: left;
+  box-align: left;
+  margin: 5px;
+}
+
+.rep-card {
+  display: flex;
+  justify-content: left;
```

Comment:
Again things are generally justified left by default so this line isn't really doing anything